scripts/generate-eslint-error-ignore-file

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# generates .eslintignore.errorfiles to list the files which have errors in,
+# so that they can be ignored in future automated linting.
+
+out=.eslintignore.errorfiles
+
+cd `dirname $0`/..
+
+echo "generating $out"
+
+{
+    cat <<EOF
+# autogenerated file: run scripts/generate-eslint-error-ignore-file to update.
+
+EOF
+
+    ./node_modules/.bin/eslint --no-ignore -f json src test |
+        jq -r '.[] | select((.errorCount) > 0) | .filePath' |
```

Comment:
Even though I didn't use this here, I'd lobby for this change (only ignore files with errors, not errors and warnings) to be used here: https://github.com/matrix-org/matrix-react-sdk/blob/87bac56b9c3b0c8688835604a5e51a00220a4e37/scripts/generate-eslint-error-ignore-file#L19

Thoughts?

Reply:
Right now lintwithexclusions in matrix-react-sdk makes it look like everything is ok when there are actually `✖ 765 problems (237 errors, 528 warnings)`. This change would at least show a bunch of those warnings so they are more visible.

Reply:
Opened https://github.com/matrix-org/matrix-react-sdk/pull/2215